data_dir/c-s_lwt_basic.yaml

```diff
@@ -0,0 +1,72 @@
+# Based on https://gist.github.com/tjake/8995058fed11d9921e31
```

Comment:
what about using the same profile as Dmitry uses for performance?

Reply:
@roydahan  It's depend if we want to validate the results or not. 
I did user profile so, that we can to validate the results of updates